ragna/deploy/_ui/modal_configuration.py

```diff
@@ -216,69 +242,82 @@ def advanced_config_ui(self):
             *(["disabled"] if disabled_assistant else []),
         ]
 
-        card = pn.Card(
-            pn.Row(
-                pn.Column(
-                    pn.pane.HTML(
-                        """<h2> Retrieval Method</h2>
-                            <span>Adjusting these settings requires re-embedding the documents, which may take some time.
-                            </span><br />
-                                         """
-                    ),
-                    pn.widgets.IntSlider.from_param(
-                        self.config.param.chunk_size,
-                        name="Chunk Size",
-                        bar_color=ui.MAIN_COLOR,
-                        css_classes=source_storage_css_classes,
-                        width_policy="max",
-                        disabled=disabled_source_storage,
-                    ),
-                    pn.widgets.IntSlider.from_param(
-                        self.config.param.chunk_overlap,
-                        name="Chunk Overlap",
-                        bar_color=ui.MAIN_COLOR,
-                        css_classes=source_storage_css_classes,
-                        width_policy="max",
-                        disabled=disabled_source_storage,
-                    ),
-                    margin=(0, 20, 0, 0),
-                    width_policy="max",
+        if is_corpus:
```

Comment:
Unfortunately explanatory text removed to make space so that everything could fit in one modal.

Reply:
Could we maybe move the definitions of these cards in two dedicated methods so we don't have an if else block that spans 70 LoC? Something like

```
card = self._make_corpus_card() if is_corpus else self._make_documents_card()
```